reinforcement_learning/rl_bin_packing_newsvendor_vehicle_routing_ray_customEnv/rl_bin_packing_ray_custom.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<p>training job run <strong>longer</strong></p><p> </p><blockquote>The sub-figures correspond to three categories of the item distribution, Perfectly Packable with Bounded Waste (BW), Perfectly Packable (PP) and Linear Waste (LW) correspondingly.&nbsp;</blockquote><p>The sub-figures correspond to three categories of the item distribution, Perfectly Packable with Bounded Waste (BW), Perfectly Packable (PP) and Linear Waste (LW) <strong>respectively</strong>.&nbsp;</p><p> </p><blockquote>You can try to replicate with a single machine with 4 GPUs and 32 CPUs for this experiment.</blockquote><p><strong>We use p3.8x instances in our experiments. </strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/934/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>